pkg/admission/prep_downscale.go

```diff
@@ -14,30 +16,29 @@ import (
 	v1 "k8s.io/api/admission/v1"
 	appsv1 "k8s.io/api/apps/v1"
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
-	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/grafana/rollout-operator/pkg/config"
-	"github.com/grafana/rollout-operator/pkg/util"
 )
 
 const (
 	PrepareDownscaleWebhookPath = "/admission/prepare-downscale"
+	extraPods                   = 3
 )
 
 func PrepareDownscale(ctx context.Context, logger log.Logger, ar v1.AdmissionReview, api *kubernetes.Clientset) *v1.AdmissionResponse {
 	client := &http.Client{
-		Timeout: 5 * time.Second,
+		Timeout: 1 * time.Second,
```

Comment:
It probably should be smaller so that we don't waste too much time waiting in the webhook. On the other hand maybe we need this much in a large busy cluster.